shuup/admin/utils/views.py

```diff
@@ -146,6 +147,12 @@ def check_and_raise_if_only_one_allowed(setting_name, obj):
 
 
 class PicotableListView(PicotableViewMixin, ListView):
+
+    def __init__(self):
+        super(PicotableListView, self).__init__()
+        self.settings = ViewSettings(self.model, self.default_columns)
+        self.columns = self.settings.columns
```

Comment:
Not so various fix since this enables the whole deal? This change probably should be introduced before the actual columns->default_columns changes. Also the default_columns should probably be introduced in base class since it's required.